### PR TITLE
New package: slim-void-theme-1.0.0

### PR DESCRIPTION
**Pull request by DirectorX**

### CODE REVIEW

File: srcpkgs/slim-void-theme/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+revision=1
+noarch=yes
+depends="slim"
+short_desc="Fancy SLiM theme for Void Linux"
+maintainer="DirectorX <DirectorX@users.noreply.github.com>"
+license="GPL-3"
+homepage="https://github.com/DirectorX/slim-void-theme"
+distfiles="https://github.com/DirectorX/${pkgname}/archive/v${version}.tar.gz"
+checksum=73b1506c6619014d9e54fd30fa82336f12ee34741eae283df7bdfd3559215c54
+wrksrc="${pkgname}-${version}"
```
Comment (Vaelatern) [inline srcpkgs/slim-void-theme/template]: Not necessary.

File: srcpkgs/slim-void-theme/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+pkgname=slim-void-theme
+version=1.0.0
+revision=1
+noarch=yes
+depends="slim"
+short_desc="Fancy SLiM theme for Void Linux"
+maintainer="DirectorX <DirectorX@users.noreply.github.com>"
+license="GPL-3"
+homepage="https://github.com/DirectorX/slim-void-theme"
+distfiles="https://github.com/DirectorX/${pkgname}/archive/v${version}.tar.gz"
```
Comment (Vaelatern) [inline srcpkgs/slim-void-theme/template]: Can we use `${homepage}`?

Comment (DirectorX) [inline srcpkgs/slim-void-theme/template]: @Vaelatern sure thing

File: srcpkgs/slim-void-theme/template (hunk 1 above)
Comment (DirectorX) [inline srcpkgs/slim-void-theme/template]: @Vaelatern  dropped

**Comment by Vaelatern** [comment]
`build_style=fetch`

**Pull request merged**